$NetBSD: patch-configure.in,v 1.1 2012/11/30 14:28:55 gdt Exp $

Make sure libpscslite.so is not linked against -lpthread on NetBSD, as
loading libpthread.so through dlopen() is now forbidden.

This should be addressed upstream, but may be controversial.

--- ./configure.in.orig	2012-10-24 06:33:56.000000000 +0200
+++ ./configure.in	2012-10-24 06:36:19.000000000 +0200
@@ -75,8 +75,22 @@
 
 AC_SUBST(PTHREAD_LIBS)
 AC_SUBST(PTHREAD_CFLAGS)
 
+case `uname` in
+NetBSD)
+	LIBPCSC_PTHREAD_LIBS=''
+	LIBPCSC_PTHREAD_CFLAGS=''
+	;;
+*)
+	LIBPCSC_PTHREAD_LIBS=$PTHREAD_LIBS
+	LIBPCSC_PTHREAD_CFLAGS=$PTHREAD_CFLAGS
+	;;
+esac
+
+AC_SUBST(LIBPCSC_PTHREAD_LIBS)
+AC_SUBST(LIBPCSC_PTHREAD_CFLAGS)
+
 # Add libtool support
 AM_PROG_LIBTOOL
 # Automatically update the libtool script if it becomes out-of-date
 #AC_SUBST(LIBTOOL_DEPS)
